app/experimenter/templates/experiments/edit_variants.html

```diff
@@ -149,4 +159,26 @@ <h4><span class="fas fa-info-circle"></span> Population</h4>
       });
     });
   </script>
+
+  {% comment %}
+  <script src="{% static "lib/bootstrap-multiselect/js/bootstrap-multiselect.js" %}"></script>
+  <script>
+    jQuery(function($) {
+      $('select[multiple]').multiselect();
+    });
+  </script>
+  {% endcomment %}
+  <script src="{% static "lib/bootstrap-select/js/bootstrap-select.min.js" %}"></script>
+  <script>
+    jQuery(function($) {
+      /* The Django form that parses locales is pretty smart. If you
+         send it ['__all__', 'sv'] it knows to set all_locales=True, locales=[],
+         etc.
+         We could attempt to replicate some of this functionality here with
+         events on $.selectpicker() but it's non-trivial and fragile.
```

Comment:
I spent a fair amount of time trying to wrangle the `.selectpicker()` events and methods so as to, for example, unselect everything else if "ALL" is chosen. And, another example, if you *had* "ALL" selected but now select "Swedish" it should unselect "ALL". 
Both were fragile and never really worked well. I felt it was better to take a break. The Django form already fixes these things. We can revisit it later.